src/proto/FunctionRpc.proto

```diff
@@ -5,6 +5,7 @@ option java_multiple_files = true;
 option java_package = "com.microsoft.azure.functions.rpc.messages";
 option java_outer_classname = "FunctionProto";
 option csharp_namespace = "Microsoft.Azure.WebJobs.Script.Grpc.Messages";
+option go_package ="rpc";
```

Comment:
rpc is very generic. Can we it more descriptive similar to other languages: com.microsoft.azure.functions.rpc.messages?

Reply:
To clarify the package name will be declared  [package rpc](https://github.com/Azure/azure-functions-go-worker/blob/dev/internal/rpc/FunctionRpc.pb.go#L4)
but when used it needs to be used with the [full path](https://github.com/Azure/azure-functions-go-worker/blob/dev/internal/worker/worker.go#L4)

So regardless how it is named here, it will be the same thing in the generated code and its usage.

if this specifically for the name here to be descriptive we can use "github.com/Azure/azure-functions-go-worker/internal/rpc" which will have the same effect. Let me know please

Reply:
yes. Lets use the full path. Can we rename the folder from "internal" to "src"?

Reply:
no, thats a total anti-pattern to have src in the package path. I'll update to the full path.